vnext/Shared/HermesRuntimeHolder.cpp

```diff
@@ -132,5 +132,33 @@ void HermesRuntimeHolder::initRuntime() noexcept {
   errorPrototype.setProperty(*m_hermesRuntime, "jsEngine", "hermes");
 }
 
+#ifdef HERMES_ENABLE_DEBUGGER
+
+std::string HermesRuntimeHolder::getDebugTargetName(const DevSettings& devSettings)  noexcept {
+  std::stringstream ss;
+
+  if (!devSettings.debuggerRuntimeName.empty()) {
+    ss << devSettings.debuggerRuntimeName;
+  } else if (!devSettings.debugBundlePath.empty()) {
+    ss << devSettings.debugBundlePath;
+  } else {
+    ss << "Hermes React Native";
+  }
+
+  ss << " (";
+
+  char exeFileName[MAX_PATH];
+  if (DWORD moduleFileNameResult = GetModuleFileNameA(/* current process */ NULL, exeFileName, MAX_PATH); moduleFileNameResult > 0) {
```

Comment:
nullptr

Reply:
`ARRAYSIZE(exeFileName)` or `std::size(exeFileName)` instead of `MAX_PATH`.